cmd/security-agent/common/check_command.go

```diff
@@ -63,6 +63,11 @@ func runCheck(cmd *cobra.Command, confPath *string, args []string) error {
 		return err
 	}
 
+	err = common.SetupConfig(*confPath)
```

Comment:
shouldn't we call it after `config.Datadog.SetConfigName` (and before `agent.WaitGetNodeLabels()`) ? Asking because for the DCA's commands we always call `SetConfigName` before `SetupConfig`  wondering if there is a good reason behind

Reply:
You're totally right, good catch!